index.html

```diff
@@ -7,6 +7,12 @@
   <title>Git Collab Lesson!</title>
 </head>
 <body>
-  
+  <header>
+    <p>This is the title of my cool page! 🤘🏽</p>
```

Comment:
This seems like a title for the page, maybe it should be a heading tag like `<h1>` instead of a `<p>` tag?

Reply:
Excellent feedback! Thanks, Susan!